concurrent/src/main/scala/tofu/concurrent/Daemon.scala

```diff
@@ -88,11 +88,11 @@ object Daemon extends DaemonInstances {
     }
   }
 
-  implicit class DaemonOps[F[_], E, A](val self: Daemon[F, E, A]) extends AnyVal {
+  implicit final class DaemonOps[F[_], E, A](private val self: Daemon[F, E, A]) extends AnyVal {
```

Comment:
What it actually does. I've thought whole `AnyVal` buisness here is not allocate any Ops, so what difference `final` modifier does?

Reply:
final is for the better chance of inlining

Reply:
But those operations are never exposed as methods of classes, they should be inlined by the compiler itself